### PR TITLE
Feed fetching script deployment

### PR DESCRIPTION
**Pull request by karpp**

### CODE REVIEW

File: capstone/feed-fetching/upload.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+scp -r ../feed-fetching wazuh-agent:~/cron/feed-fetching
+ssh wazuh-agent << EOF
```
Comment (niedziol) [inline capstone/feed-fetching/upload.sh:2]: Explain ssh config that one must have for this line to work.

File: capstone/feed-fetching/upload.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    cd ~/cron/feed-fetching
+    python3 -m virtualenv venv
+    source venv/bin/activate
```
Comment (niedziol) [inline capstone/feed-fetching/upload.sh:5]: And what if the virtualenv is already created and we only want to update the script?

Comment (karpp) [reply capstone/feed-fetching/upload.sh:5]: Then it will leave existing virtualenv and install only new libraries from requirements

**Review by niedziol — APPROVED** [review]
LGTM

**Review by mgmacias95 — APPROVED** [review]
LGTM

**Pull request merged**